softperipheral/doc/sQSPI/nrf54H20_porting_v0_1_0.rst

```diff
@@ -0,0 +1,231 @@
+
+.. code-block::
+
+   nrfxlib/
+   ├──...
+   └── softperipheral/
+      ├── include
+      │   ├── softperipheral_meta.h
+      │   └── softperipheral_regif.h
+      ├── ...
+      └── sQSPI
+          ├── include
+          │   └── hal
+          │       └── nrf_qspi2.h
+          │   ├── nrf54h20
+          │   │   ├── sqspi_firmware.h
+          │   │   ├── sqspi_firmware_v0.1.0.h
+          │   │   └── ...
+          │   ├── nrf_sp_qspi.h
+          │   ├── nrfx_config_qspi2.h
+          │   ├── nrfx_qspi2.h
+          └── src
+              └── nrfx_qspi2.c
+
+.. note::
+   The main interface for sQSPI is the :file:`nrfx_qspi2.h` file.
+
+Header files
+============
+
+For an sQSPI application to function correctly, it requires specific header files used by the driver code (internal paths) and specific header files for nrfx (external dependencies).
+You must include paths to these files in the build environment's list of include paths.
+The following list is a detailed breakdown of the necessary paths:
+
+* Repository internal paths:
+
+  * :file:`softperipheral/include` - Soft peripherals register interface and metadata
+  * :file:`softperipheral/sQSPI/include` - sQSPI register interface and driver header
+  * :file:`softperipheral/sQSPI/include/nRF54H20` - sQSPI FLPR firmware
+
+* External dependencies:
+
+   * Path to nrfx to include the main nrfx header file - :file:`nrfx.h`.
+   * Path to nrfx/drivers/include, to include nrfx driver header files - For example :file:`nrfx_dppi.h`.
+   * Path to the application specific headers and macros - For example :file:`nrfx_glue.h`, :file:`nrfx_log.h` or a config header.
+     See an example configuration for the :file:`nrfx_config.h` file:
+
+      .. code-block:: c
+
+         #ifndef NRFX_CONFIG_H__
+         #define NRFX_CONFIG_H__
+
+         #include "softperipheral_regif.h" // To Resolve correct VPR IRQn for the SoC.
+         #define nrfx_qspi2_irq_handler        SP_VPR_IRQHandler
+
+         #define NRFX_QSPI2_ENABLED            (1)
+         #define NRFX_QSPI2_MAX_NUM_DATA_LINES (4)
+
+         #endif // NRFX_CONFIG_H__
+
+Source files
+============
+
+For an sQSPI application to function properly, it is essential to compile the driver implementation from the source file :file:`nrfx_qspi2.c`.
+
+Application Core and FLPR configuration
+***************************************
+
+For an sQSPI application to run properly, nRF54H20 must run at highest base clock frequency. The user/integrator should take care of this.
+
+.. note::
+   In any of the following settings, is assumed that the user/integrator has allocated memory for the data pointers used by the sQSPI driver, this memory space must be accessible by both Application Core and FLPR.
+
+   This allocated memory is independent from the one listed in :ref:`nrf54H20_porting_guide_ram_configuration`, the intention is for it to serve as shared memory for communication rather than containing the sQSPI executable code.
```

Comment:
```suggestion
   The memory is independent from the one outlined in the :ref:`nrf54H20_porting_guide_ram_configuration` subsection. 
   It is designated to function as shared memory for communication purposes, rather than to store the sQSPI executable code.
```